docs/modules/operation/pages/deep-dive/service-assurance/path-outages.adoc

```diff
@@ -59,6 +58,6 @@ In the Web UI go to menu:Gear Menu[Manage Provisioning Requisitions > Edit the R
 | node-04
 |===
 
-IMPORTANT: The IP Interface which is set to Primary is selected as the Critical IP.
-           In this example, it is important that the IP interface on default-gw-01 in the network 192.168.1.0/24 is set as Primary interface.
-           The IP interface in the network 172.23.42.0/24 on default-gw-02 is set as Primary interface.
+IMPORTANT: The IP interface, set to primary is selected as the critical IP.
+In this example, it is important that the IP interface on `default-gw-01` in the network `192.168.1.0/24` is set as primary interface.
+The IP interface in the network `172.23.42.0/24` on `default-gw-02` is set as primary interface.
```

Comment:
Is this separate from the above statement? A second critical path?

Reply:
Trying to parse this. @mmahacek could you take a closer look? I am not sure what we are trying to accomplish here, and I think that there may be some explanatory text missing.

Reply:
Jeff G. confirmed that while this could be better written, it is correct. Basically it's so that gw-02 is also covered as part of the critical path.